app/data_transfer/dayone/mappers.py

```
@@ -0,0 +1,524 @@
+            raw_dayone.pop("videos", None)
+
+        return {
+            "source": "dayone",
+            "raw_dayone": raw_dayone,
+        }
+
+    @staticmethod
+    def _map_media_common(
+        media_path: Path,
+        identifier: str,
+        entry_external_id: str,
```

Ruff (0.14.10)

Unused static method argument: `entry_external_id`

(ARG004)